netcore/Makefile

```diff
@@ -234,6 +239,36 @@ build-test-corefx-%:
 xunit-summary:
 	./xunit-summary.py "corefx/tests"
 
+
+update-tests-coreclr: coreclr/.stamp-tests
+
+coreclr/.stamp-tests:
+	rm -rf coreclr/tests
```

Comment:
The `coreclr` directory is never created on my machine and all the successive step consequently fail.